internal/core/data/config/config.go

```diff
@@ -40,10 +41,14 @@ type WritableInfo struct {
 }
 
 type ReadingRetention struct {
-	Enabled  bool
+	Interval        string
+	DefaultMaxCap   int64
+	DefaultMinCap   int64
+	DefaultDuration string
+}
+
+type AutoEventRetention struct {
```

Comment:
```suggestion
type EventRetention struct {
```

Reply:
Rename ReadingRetention to EventRetention.